src/main/java/core/basesyntax/service/implementation/CreateReportServiceImplementation.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.implementation;
+
+import core.basesyntax.service.CreateReportService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CreateReportServiceImplementation implements CreateReportService {
+    private static final int START_INDEX = 3;
+    private OperationStrategy operationStrategy;
+
+    public CreateReportServiceImplementation(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
+    public String createReport(String dataFromFile) {

```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)